pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -157,9 +157,9 @@ func (pl *VolumeBinding) PreBind(ctx context.Context, cs *framework.CycleState,
 	return nil
 }
 
-// Unreserve clears pod binding state.
-// TODO(#90962) Revert assumed PV/PVC cache
+// Unreserve clears assumed PV and PVC cache and pod binding state.
```

Comment:
@cofyc this function needs to be idempotent, meaning ```Unreserve``` could be called multiple times for the same pod in the same cycle, it could also be called when ```Reserve``` wasn't.

Reply:
I think this function is idempotent.

Reply:
it is idempotent as it does nothing if no cache or binding state found

Reply:
@mrkm4ntr maybe it's better to clarify it in the comment, e.g. It's idempotent, and does nothing if no cache or binding state found for the given pod.

Reply:
OK!